packages/utilities/src/utilities/smoothScroll/smoothScroll.js

```diff
@@ -6,12 +6,12 @@
  */
 
 /**
- * Utility handles smoothScroll to the target element after OnClick
+ * Utility handles smoothScroll to the target element after onMouseDown
```

Comment:
Wonder if we should explicitly say this is for anchor links?

Reply:
updated.